Policies/github-privacy-statement.md

```diff
@@ -238,10 +243,10 @@ In addition, if your account has private repositories, you control the access to
 Github will provide notice regarding private repository access where not prohibited by law or if in response to a security threat or other risk to security.
 
```

Comment:
How much they pay you to shill for Microsoft , this is how it beings , this is how they killed Cent OS, this is how they jeopardized java, corporations are never to trust with good products.

—
Reply to this email directly, [view it on GitHub](https://github.com/github/site-policy/pull/582#issuecomment-1203617810), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/AYP5MYNULHE65TOOMN7J6VTVXIROHANCNFSM55MKCXWQ).
You are receiving this because you are subscribed to this thread.
Use uBlock Origin as it says if you really can't stand a few cookies

Do you know about Manifest version 3 ? They are going to kill Ublock Origin , how stupid you thing we are ?

—
Reply to this email directly, [view it on GitHub](https://github.com/github/site-policy/pull/582#issuecomment-1203620308), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/AYP5MYIC5BK6OGMQYTIDDPTVXIRXNANCNFSM55MKCXWQ).
You are receiving this because you are subscribed to this thread.
Why are people getting so riled up when this change only impacts the Enterprise marketing subdomains? Makes no sense to me

How much they pay you to shill for Microsoft , this is how it beings , this is how they killed Cent OS, this is how they jeopardized java, corporations are never to trust with good products.

Cent OS wasn't even a Microsoft product, it was a product owned by Red Hat, which is in turn a subsidiary of IBM. Microsoft had nothing to do with why they killed off the standard CentOS operating system. And I'm not shilling for Microsoft at all, I'm simply trying to be reasonable here. Not everything that happens is Microsoft's fault, and if you think that, then you are immediately wrong.

And Microsoft never jeopardized Java - Java is Oracle, and Java is still going strong and being used in lots of software and products to this day, including Android. Microosft has done nothing to Java, so that last point is moot, null, and void.

Do you know about Manifest version 3 ? They are going to kill Ublock Origin , how stupid you think we are ?

That is Google's problem, not Microsoft's. Again, Manifest v3 is a Google-developed feature, not something developed by Microsoft. Microsoft does use Chromium, but this is Google's fault, not Microsoft's. People love bashing Microsoft for things they never even do, as if everything wrong that ever happens in the world is Microsoft's fault, which is not the case at all.

—
Reply to this email directly, [view it on GitHub](https://github.com/github/site-policy/pull/582#issuecomment-1203643899), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/AYP5MYP2EVKHLSKU5SG6XY3VXIUPRANCNFSM55MKCXWQ).
You are receiving this because you are subscribed to this thread.
Cent OS wasn't even a Microsoft product, it was a product owned by Red Hat, which is in turn a subsidiary of IBM. Microsoft had nothing to do with why they killed off the standard CentOS operating system. And I'm not shilling for Microsoft at all, I'm simply trying to be reasonable here. Not everything that happens is Microsoft's fault, and if you think that, then you are immediately wrong. And Microsoft never jeopardized Java - Java is Oracle, and Java is still going strong and being used in lots of software and products to this day, including Android. Microsoft has done nothing to Java, so that last point is moot, null, and void.

That is Google's problem, not Microsoft's. Again, Manifest v3 is a Google-developed feature, not something developed by Microsoft. Microsoft does use Chromium, but this is Google's fault, not Microsoft's. People love bashing Microsoft for things they never even do, as if everything wrong that ever happens in the world is Microsoft's fault, which is not the case at all.

Microsoft, Google, Aamzon, Red Hat, these are corporations, they all have same motive: Money, I was simply stating what happened to Cent OS, Java, Manifest v2 will happen to Github because Microsoft being corporation will burn github down for monetary gains.

I have deleted my comment of you accusing of payment and shilling because that was somewhat in bad taste

—
Reply to this email directly, [view it on GitHub](https://github.com/github/site-policy/pull/582#issuecomment-1203648188), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/AYP5MYINKH4RMXR5ACARMLLVXIU7HANCNFSM55MKCXWQ).
You are receiving this because you are subscribed to this thread.
Cent OS wasn't even a Microsoft product, it was a product owned by Red Hat, which is in turn a subsidiary of IBM. Microsoft had nothing to do with why they killed off the standard CentOS operating system. And I'm not shilling for Microsoft at all, I'm simply trying to be reasonable here. Not everything that happens is Microsoft's fault, and if you think that, then you are immediately wrong. And Microsoft never jeopardized Java - Java is Oracle, and Java is still going strong and being used in lots of software and products to this day, including Android. Microsoft has done nothing to Java, so that last point is moot, null, and void.

That is Google's problem, not Microsoft's. Again, Manifest v3 is a Google-developed feature, not something developed by Microsoft. Microsoft does use Chromium, but this is Google's fault, not Microsoft's. People love bashing Microsoft for things they never even do, as if everything wrong that ever happens in the world is Microsoft's fault, which is not the case at all.

Microsoft, Google, Aamzon, Red Hat, these are corporations, they all have same motive: Money, I was simply stating what happened to Cent OS, Java, Manifest v2 will happen to Github because Microsoft being corporation will burn github down for monetary gains.

I have deleted my comment of you accusing of payment and shilling because that was somewhat in bad taste

Everything in this world requires money. And that's what matters. No money = No everything.

Money may not buy everything. But money can buy almost anything.

And I think you're hating their organization.

—
Reply to this email directly, [view it on GitHub](https://github.com/github/site-policy/pull/582#issuecomment-1203674494), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/AYP5MYJJFYCWKZYS2YN7AETVXIX7ZANCNFSM55MKCXWQ).
You are receiving this because you are subscribed to this thread.
https://gitea.io/

—
Reply to this email directly, [view it on GitHub](https://github.com/github/site-policy/pull/582#issuecomment-1203675458), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/AYP5MYPVBVPKAN7UAHUCPCLVXIYDDANCNFSM55MKCXWQ).
You are receiving this because you are subscribed to this thread.
[@microsoft](https://github.com/microsoft)
9440ec208f21f3a862f501af90003ef4

—
Reply to this email directly, [view it on GitHub](https://github.com/github/site-policy/pull/582#issuecomment-1203687789), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/AYP5MYOB5XHYQ342GFTZQ33VXIZNHANCNFSM55MKCXWQ).
You are receiving this because you are subscribed to this thread.
@mxrcury approved this pull request.

—
Reply to this email directly, [view it on GitHub](https://github.com/github/site-policy/pull/582#pullrequestreview-1060007611), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/AYP5MYKNAWM4PRP33XA2OW3VXI5OTANCNFSM55MKCXWQ).
You are receiving this because you are subscribed to this thread.